Opentrons/opentrons
# fix(app): add cursor and pageLength to GET /commandErrors

# Overview

closes [RQA-3031](https://opentrons.atlassian.net/browse/RQA-3031) and [RQA-3047](https://opentrons.atlassian.net/browse/RQA-3047).
pass in cursor and page length to get run command errors. 

## Test Plan and Hands on Testing

- upload a protocol that will enter ER.
- make sure there are a few errors to show.
- at the end of the run make sure when you click 'view error details' is showing the full error list.

## Changelog

added cursor and pageLength to GET request.

## Risk assessment

None.

[RQA-3031]: https://opentrons.atlassian.net/browse/RQA-3031?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[RQA-3047]: https://opentrons.atlassian.net/browse/RQA-3047?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
